README.md

```diff
@@ -1 +1,28 @@
-# hello-microverse
\ No newline at end of file
+hello-microverse
+
+First microverse project. Using branches, linters, .gitignore, html, css and Git Bash.
+
+Built With:
+Git Bash
+VS Code
+Git
+
+Authors
+👤 Botlhale Setou
+
+GitHub: @Botlhale-Setou
+Twitter: @Botlhale_Setou
+LinkedIn: https://www.linkedin.com/in/botlhalesetou/
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
```

Comment:
- [Optional] Kindly consider adding steps to get a copy of your project locally to your readme file. This will make your work more professional.
- Also consider adding spaces and making your headings bold just as it is in [this](https://github.com/microverseinc/readme-template) template provided by microverse.